aptos-move/framework/aptos-stdlib/sources/math128.move

```diff
@@ -1,6 +1,9 @@
 /// Standard math utilities missing in the Move Language.
 module aptos_std::math128 {
 
+    /// Abort value when an invalid argument is provided.
```

Comment:
I think we should use separate error codes if possible instead of EINVALID_ARG for all invalid arguments case. Specifically, let's use a different error code for different functions so we can make the error messages easy to understand

Reply:
I renamed the error codes.